admin_solutions/master_node_config.adoc

```diff
@@ -0,0 +1,409 @@
+
+- `*BootstrapPolicyFile*` (string): Points to a template that contains roles and rolebindings that will be created if no policy object exists in the master namespace.
+- `*OpenShiftSharedResourcesNamespace*` (string): The namespace where shared OpenShift resources are located, such as shared templates.
+- `*OpenShiftInfrastructureNamespace*` (string): The namespace where OpenShift infrastructure resources are located, such as controller service accounts.
+
+|`*projectConfig*`
+a|Holds information about project creation and defaults:
+
+- `*DefaultNodeSelector*` (string): Holds the default project node label selector.
+- `*ProjectRequestMessage*` (string): The string presented to a user if they are unable to request a project via the projectrequest API endpoint.
+- `*ProjectRequestTemplate*` (string): The template to use for creating projects in response to projectrequest. It is in the format `<namespace>/<template>`. It is optional, and if it is not specified, a default template is used.
+- `*SecurityAllocator*`: Controls the automatic allocation of UIDs and MCS labels to a project. If nil, allocation is disabled:
+  * `*mcsAllocatorRange*` (string): Defines the range of MCS categories that will be assigned to namespaces. The format is `<prefix>/<numberOfLabels>[,<maxCategory>]`. The default is `s0/2` and will allocate from c0 -> c1023, which means a total of 535k labels are available. If this value is changed after startup, new projects may receive labels that are already allocated to other projects. The prefix may be any valid SELinux set of terms (including user, role, and type). However, leaving the prefix at its default allows the server to set them automatically. For example, `s0:/2` would allocate labels from s0:c0,c0 to s0:c511,c511 whereas `s0:/2,512` would allocate labels from s0:c0,c0,c0 to s0:c511,c511,511.
+  * `*mcsLabelsPerProject*` (integer): Defines the number of labels to reserve per project. The default is `5` to match the default UID and MCS ranges.
+  * `*uidAllocatorRange*` (string): Defines the total set of Unix user IDs (UIDs) automatically allocated to projects, and the size of the block each namespace gets. For example, `1000-1999/10` would allocate ten UIDs per namespace, and would be able to allocate up to 100 blocks before running out of space. The default is to allocate from 1 billion to 2 billion in 10k blocks, which is the expected size of ranges for Docker images when user namespaces are started.
+
+|`*routingConfig*`
+a|Holds information about routing and route generation:
+
+- `*Subdomain*` (string): The suffix appended to $service.$namespace. to form the default route hostname. Can be controlled via Ansible with `*openshift_master_default_subdomain*`. Example:
++
+----
+routingConfig:
+  subdomain:  ""
+----
+
+|`*networkConfig*`
+a|To be passed to the compiled in network plug-in. Many of the options here can be controlled in the Ansible inventory.
```

Comment:
There a hyphen missing here?

compiled-in-network

compiled-in network

compiled in-network

?